### PR TITLE
PR for contact form page

### PR DESCRIPTION
**Pull request by Tekhlay**
## Accomplished tasks 🌍 
- Add one contact form page to the existing one.
- Modified the CSS part of the page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -190,10 +190,10 @@ <h2>I'm always interested in hearing about new projects, so if you'd like to cha
...
         <form action="#" class="footer-form">
-            <input class="input" type="text" placeholder="Full name">
-            <input class="input" type="email" placeholder="Email address">
-            <textarea name="" id="" cols="30" rows="10">Enter text here</textarea>
-            <button class="abt-btn" type="button">Get in touch</button>
+            <input class="input" type="text" placeholder="Full name" maxlength="30" required>
+            <input class="input" type="email" placeholder="Email address" required>
+            <textarea name="" id="" cols="30" rows="10" maxlength="500">Enter text here</textarea>
+            <button class="abt-btn" type="submit">Get in touch</button>
         </form>
```
Comment (1GORDON) [inline index.html:197]: Great job @Tekhlay 🤝

- [ ] Kindly make sure the form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html). `action` value should be similar to `https://formspree.io/f/{form_id}` and `method` value equals `POST`.
- [ ] Kindly make the `Enter text here` a placeholder for the `Textarea` field.
- [ ] You may also add the `required` attribute to the `Textarea` field.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Tekhlay  ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sajid-munawar — CHANGES_REQUESTED** [review]
Hi @Tekhlay , 

_Great job so far 👏 💯 👍. You've done well implementing some of the requested changes. However, there are still some which aren't addressed yet. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 
  - Addressed some of the issues raised from the previous review. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
  
  - N/A 

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by omarsalem7 — APPROVED** [review]
Hi @Tekhlay 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**